.github/workflows/validate_schemas.yml

```diff
@@ -19,7 +19,7 @@ jobs:
         run: |
           echo "Finding YAML and JSON files with a \$schema property…"
           files=$(grep -rl "\$schema:" . --include="*.yaml" --include="*.json" --exclude-dir="node_modules" | paste -sd '|' -)
-          json_files=$(grep -rl "\"\$schema\":" . --include="*.json" --exclude-dir="node_modules" | paste -sd '|' -)
+          json_files=$(grep -rl "\"\$schema\":" . --include="*.json" --exclude-dir="node_modules" | grep -v "context7.json" | paste -sd '|' -)
```

Comment:
Ok, so the context7 json schema is too strict, and doesn't allow the `$schema` directive, lol https://github.com/upstash/context7/issues/377.  Skip that for now